sdk/tests/ClientTest.cc

```diff
@@ -0,0 +1,69 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include "Client.h"
+#include "AccountId.h"
+#include "ED25519PrivateKey.h"
+#include "Hbar.h"
+
+#include <gtest/gtest.h>
+
+using namespace Hedera;
+
+class ClientTest : public ::testing::Test
+{
+};
+
+TEST_F(ClientTest, ConstructClient)
+{
+  Client client;
+  EXPECT_EQ(client.getOperator(), nullptr);
+  EXPECT_EQ(client.getDefaultMaxTransactionFee(), nullptr);
+  EXPECT_EQ(client.getRequestTimeout(), std::chrono::minutes(2));
+}
+
+TEST_F(ClientTest, SetOperator)
```

Comment:
Should we test both versions of SetOperator?

Reply:
I wasn't sure if I could, but I just tried it with a `std::move()` and it worked, so yeah I'll add that.